buildSrc/src/main/groovy/Dependencies.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (c) 2019 Connexta, LLC
+
+Released under the GNU Lesser General Public License version 3; see
+https://www.gnu.org/licenses/lgpl-3.0.html
+*/
+// Default Package
+class Dependencies {
```

Comment:
✏️ Maybe split this into three classes: `Dependencies`, `TestDependencies`, and `PluginDependencies` so that it is obvious when we use them, e.g. `testImplementation(TestDependencies.springBootTestStarter())`.

Reply:
I'm not sure we need to be that obvious about dependencies we're using, I think having the `testImplementation` should be enough